candi/bashible/common-steps/node-group/063_install_kubelet_forker.sh.tpl

```diff
@@ -41,5 +43,14 @@ until ss -nltp4 | grep -qE "127.0.0.1:10248.*pid=$CHILDREN_PID" && curl -s -f ht
   echo "d8-kubelet-forker [INFO] Waiting for HTTP 200 response from /healthz endpoing of kubelet with PID $CHILDREN_PID (attempt $attempt of $max_attempts)..."
   sleep 1
 done
+
+
+{{- if hasKey .nodeGroup "fencing" }}
+  {{ if eq .nodeGroup.fencing.mode "Watchdog" }}
+# fencing settings
+sysctl -w kernel.panic=0
+  {{- end }}
+{{- end }}
+
```

Comment:
rewrite it as in the previous file please

Reply:
fixed